### PR TITLE
feat: add landing page

### PR DESCRIPTION
**Pull request by spaenleh**
This PR is based on top of #433

### CODE REVIEW

File: src/modules/landing/home/TitleSection.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+              alignItems="center"
+            >
+              <EPFL width="150px" height="3rem" />
+              <BeLEARN width="150px" height="3rem" />
+              <EdTech width="150px" height="3rem" />
+              <Unine width="150px" height="3rem" />
+              <GoLab width="150px" height="3rem" />
+              {/* <span>GOGA</span>
+              <span>IHUB4SCHOOL</span>
+              <span>UNCTAD</span> */}
```
Comment (pyphilia) [inline src/modules/landing/home/TitleSection.tsx:102]: Remove?

File: src/modules/landing/home/icons/EdTech.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+export function EdTech({
+  width,
+  height,
+}: {
+  width: string;
+  height: string;
+}): JSX.Element {
+  return (
+    <svg
+      // id="logo"
```
Comment (pyphilia) [inline src/modules/landing/home/icons/EdTech.tsx:10]: Remove?

File: src/modules/landing/home/icons/EdTech.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      viewBox="-15 0 185 110.7"
+      // style="width: 110px; height: 50px; fill: rgb(238, 238, 238);"
```
Comment (pyphilia) [inline src/modules/landing/home/icons/EdTech.tsx:12]: Remove?

File: src/modules/landing/home/icons/Unine.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+}: {
+  width: string;
+  height: string;
+}): JSX.Element {
+  return (
+    <svg
+      version="1.1"
+      id="svg2"
+      // width={width}
+      // height={height}
```
Comment (pyphilia) [inline src/modules/landing/home/icons/Unine.tsx:13]: Remove?

File: src/routes/index.lazy.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  component: Index,
+});
+
+function Index() {
+  const { isAuthenticated } = useAuth();
+  const { isMobile } = useMobileView();
+  const { fill: primary } = useButtonColor('primary');
+
+  return (
+    <Stack alignItems="center" id="pageWrapper">
```
Comment (pyphilia) [inline src/routes/index.lazy.tsx:32]: Do you need the id?

File: src/routes/index.lazy.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+      <Stack
+        id="navBar"
```
Comment (pyphilia) [inline src/routes/index.lazy.tsx:34]: same.

**Review by pyphilia — APPROVED** [review]
Thanks for this PR!

File: src/modules/landing/footer/Footer.tsx [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+  },
+  {
+    title: 'Twitter',
+    href: 'https://twitter.com/graasp',
+    Icon: TwitterIcon,
+  },
+  {
+    title: 'Instragram',
+    // TODO: this link needs to be updated
+    href: 'https://www.facebook.com/graasp',
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply src/modules/landing/footer/Footer.tsx]: Thank you !

File: src/modules/landing/home/TitleSection.tsx (hunk 1 above)
Comment (spaenleh) [reply src/modules/landing/home/TitleSection.tsx:102]: Should we remove them completely ? I did not find nice svg logos for them, only slightly bad jpeg images.

**Pull request merged**